### PR TITLE
[S] RR: add recomend % and stars with ratings bar feature

### PR DESCRIPTION
**Pull request by squeakypickles33**
Made the bars ratings split by percentage. Small thing I need to do is add the percentage of reviews

### CODE REVIEW

**Review by yuxinlu1 — COMMENTED** [review]
I don't think you need to write the specific ID number, because I have already updated the main app.jsx.

File: client/src/components/RatingsAndReviews/Sorting.jsx [hunk 1]
```diff
@@ -4,13 +4,13 @@ import ReviewsList from "./ReviewsList"
...
   return (
-    <>
-    <select value={sort} onChange={(event) => setSort(event.target.value)}>
-      <option value="relevant">relevance</option>
-      <option value="helpful">helpfulness</option>
-      <option value="newest">newest</option>
-    </select>
-    </>
+    <div>
+      <select value={sort} onChange={(event) => setSort(event.target.value)}>
```
Comment (burhan-syed) [inline client/src/components/RatingsAndReviews/Sorting.jsx:8]: This will need a label for accessibility

**Review by burhan-syed — APPROVED** [review]
Overall looks good to merge. 
Be sure to include a label with your select to pass accessibility requirements.
Also consider changing your px units to rem for consistency. 

For the merge conflict be sure to accept the existing changes in main.

File: client/src/components/RatingsAndReviews/ProductBreakdown.jsx
Comment (yuxinlu1) [inline client/src/components/RatingsAndReviews/ProductBreakdown.jsx:1]: If the ratings object is empty or if all the rating counts are zero, there would be a division by zero when calculating the average

File: client/src/components/RatingsAndReviews/RatingBars.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const RatingBar = ({ stars, count, totalReviews }) => {
+
+  const greenBarWidth = `${(count / totalReviews) * 100}%`;
+  const grayBarWidth = `${(count / totalReviews) * 100}%`;
```
Comment (yuxinlu1) [inline client/src/components/RatingsAndReviews/RatingBars.jsx:6]: you can delete grayBar

**Pull request merged**